### PR TITLE
Milestone-02: Mobile-version .

### PR DESCRIPTION
**Pull request by jenadiusnicholaus**
In this version added .
     > Recent and list of  works done design
     > About me design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,40 +12,289 @@
...
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
+                    </div>
+                </div>
+            </div>
+        </section>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly use  `display:grid` for the main container of the projects, it is required you use grid to display projects in the section
- [ ]

File: index.html [hunk 2]
```diff
@@ -12,40 +12,289 @@
...
+                    <ul class="ul-teck-stacks">
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">css </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">html </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">Bootstrap </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">Ruby </button></li>
+                    </ul>
+                </div>
+                <div class="btn-see-project">
+                    <button type="button" class="">See Project </button>
+                </div>
```
Comment (prolajumokeoni) [inline index.html]: Figma design           |  Your design
:-------------------------:|:-------------------------:
image |  hisworks

- [ ] Kindly use the appropriate font colors, and sizes for these content check  screenshot above ⬆️ 
- [ ] Kindly add hover effect to button

File: index.html [hunk 3]
```diff
@@ -12,40 +12,289 @@
...
+
+        <!-- About me -->
+        <section id="about-me" class="about-me ">
+            <div class="intro">
+                <h1>About me</h1>
+                <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work
+                    and
+                    experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <button class="btn-get-my-resume" type="button">Get my resume</button>
```
Comment (prolajumokeoni) [inline index.html:249]: - [ ] Kindly remove border from the button so it looks like the button in the Figma design

File: index.html [hunk 4]
```diff
@@ -12,40 +12,289 @@
...
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

File: index.html [hunk 5]
```diff
@@ -12,40 +12,289 @@
...
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

File: index.html [hunk 6]
```diff
@@ -12,40 +12,289 @@
...
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button  type="button" class="">See Project </button>
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

File: index.html [hunk 7]
```diff
@@ -12,40 +12,289 @@
...
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

File: index.html [hunk 8]
```diff
@@ -12,40 +12,289 @@
...
+                                <ul  class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

File: index.html [hunk 9]
```diff
@@ -12,40 +12,289 @@
...
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

File: index.html [hunk 10]
```diff
@@ -12,40 +12,289 @@
...
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

File: index.html [hunk 11]
```diff
@@ -12,40 +12,289 @@
...
```
Comment (prolajumokeoni) [inline index.html]: - [ ]  Kindly add hover effect to button so it looks like button in the figma design

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @jenadiusnicholaus , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there 💯 

## Highlights 
- Good PR 👍🏿 
- Linters are passing 👍🏿 
- Good Readme 👍🏿 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
almost-there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jenadiusnicholaus) [reply index.html]: But was thinking that  display:grid is need in  during the desktop version of tho f the app. right?

Comment (prolajumokeoni) [reply index.html]: @jenadiusnicholaus  part of the requirements is to use display :  grid for the main container wrapping all the projects

Comment (prolajumokeoni) [reply index.html]: Also each project content is to be aligned with flex box, so I strongly advise you use the display grid for the main container
You can also ask for a second opinion from the next reviewer 🤗

**Review by Kalunge — APPROVED** [review]
Hi @jenadiusnicholaus , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jenadiusnicholaus** [comment]
Thank you for your comment

**Pull request merged**